app/lib/components/term.js

```diff
@@ -149,6 +160,10 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.devicePixelRatio !== window.devicePixelRatio) {
+      this.term.prefs_.set('font-smoothing', this.getFontSmoothing());
```

Comment:
Subscribing to move sounds like a very good idea. We can subscribe to an `rpc` event and then dispatch a redux action?